.github/ISSUE_TEMPLATE/Bug_report.md

```diff
@@ -1,11 +1,18 @@
 ---
-name: 🐛 Bug Report
-about: If something isn't working as expected 🔧.
+name: "\U0001F41B Bug Report"
```

Comment:
Any way we can leave the emojis in?  This is really hard to know what you are editing.

Reply:
will do. github editor did the escape for some reason